pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1514,6 +1515,16 @@ func (og *operationGenerator) GenerateVerifyControllerAttachedVolumeFunc(
 		return volumetypes.GeneratedOperations{}, volumeToMount.GenerateErrorDetailed("VerifyControllerAttachedVolume.FindPluginBySpec failed", err)
 	}
 
+	if volumeToMount.PluginIsAttachable {
+		cachedAttachedVolumes, _ := og.volumePluginMgr.Host.GetAttachedVolumes()
```

Comment:
maybe add comment indicating this is early check attached volume from node status using nodelister to avoid back off? Later we will directly call api server to get the latest node status to confirm the VolumeAttached list to avoid race condition.

Reply:
Added a comment.